doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+    ],
+    "ftReceiptCase": 0,
+    "ftReceiptCaseData": "string",
+    "cbReceiptAmount": 0,
+    "cbUser": "string",
+    "cbArea": "string",
+    "cbCustomer": "string",
+    "cbSettlement": "string",
+    "cbPreviousReceiptReference": "string"
+  },
+  "response": {
+    "ftCashBoxID": "string",
+    "ftQueueID": "string",
+    "ftQueueItemID": "string",
+    "ftQueueRow": 0,
+    "cbTerminalID": "string",
+    "cbReceiptReference": "string",
+    "ftCashBoxIdentification": "string",
+    "ftReceiptIdentification": "string",
+    "ftReceiptMoment": "2019-08-24T14:15:22Z",
+    "ftReceiptHeader": [
+      "string"
+    ],
+    "ftChargeItems": [
+      {
+        "position": 0,
+        "quantity": 0,
+        "description": "string",
+        "amount": 0,
+        "vatRate": 0,
+        "ftChargeItemCase": 0,
+        "ftChargeItemCaseData": "string",
+        "vatAmount": 0,
+        "accountNumber": "string",
+        "costCenter": "string",
+        "productGroup": "string",
+        "productNumber": "string",
+        "productBarcode": "string",
+        "unit": "string",
+        "unitQuantity": 0,
+        "unitPrice": 0,
+        "moment": "2019-08-24T14:15:22Z"
+      }
+    ],
+    "ftChargeLines": [
+      "string"
+    ],
+    "ftPayItems": [
+      {
+        "position": 0,
+        "quantity": 0,
+        "description": "string",
+        "amount": 0,
+        "ftPayItemCase": 0,
+        "ftPayItemCaseData": "string",
+        "accountNumber": "string",
+        "costCenter": "string",
+        "moneyGroup": "string",
+        "moneyNumber": "string",
+        "moment": "2019-08-24T14:15:22Z"
+      }
+    ],
+    "ftPayLines": [
+      "string"
+    ],
+    "ftSignatures": [
+      {
+        "ftSignatureFormat": 0,
+        "ftSignatureType": 0,
+        "caption": "string",
+        "data": "string"
+      }
+    ],
+    "ftReceiptFooter": [
+      "string"
+    ],
+    "ftState": 0,
+    "ftStateData": "string"
+  }
+}
+```
+</details>
+
+**Responses:**
+
+200 - Returns a unique identifier, which can be used to obtain the result of the operation via the response endpoint.
+
+<details>
+<summary>Response sample (JSON):</summary>
+
+```
+{
+    "type": "print",
+    "identifier": "0ccf5ada-7d0d-4531-bc2c-9c602d26e4fe"
+}
+```
+</details>
+
+400 - Bad request "not supported" (Please check the request) 
+
+401 – Unauthorized (No or wrong Accesstoken or CashBoxID in header)
+
+## Return the response of a previous async call (Response endpoint) 
+
+This method is used to obtain the result of a previously executed asynchronous operation. Callers should pass the result object from this referenced operation into the body, and the method will either return the requested response, or HTTP 204 in case the operation has not finished yet.
+
+The response type of this methos depends on the type of the referenced asynchronous operation. For the digital receipt the PrintResponse can be used to obtain the URL to the digital receipt:
```

Comment:
I think you're missing the URL and the required headers of the response endpoint here, right?